### PR TITLE
Find quiz

### PR DESCRIPTION
**Pull request by bhgottfried**
Create a quiz that users must fill out in order to generate restaurant suggestions.

Users will receive one recommendation at a time until they accept it.

After each accept or rejection, the restaurant ID is passed to the backend so in the future, we can use it to train the user's recommendation model to give them more tailored suggestions.

### CODE REVIEW

**Review by jhyndma — APPROVED** [review]
FindQuiz correctly queries backed and sends data when necessary when a quiz is submitted. Approved

File: frontend/src/FindQuiz.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+                            <option value="$">$</option>
+                            <option value="$$">$$</option>
+                            <option value="$$$">$$$</option>
+                            <option value="$$$$">$$$$</option>
+                        </select>
+                    </label>
+                    <br />
+                    <label id="q3">
+                        3. Maxmimum Distance (mi.)&emsp;
+                        <input type="number" defaultValue="1" min="1" value={this.state.q2.value} name="q3"
```
Comment (EliasTalcott) [inline frontend/src/FindQuiz.js:152]: We may want to set this to be a maximum of 24 or 25 miles. Yelp API won't return results if the distance is over 40km (24.8 miles), and reducing the radius to 40km in the backend may be confusing to users (e.g. looking for a restaurant 50 miles away and nothing shows up). Quote from [Yelp API Documentation](https://www.yelp.com/developers/documentation/v3/business_search):

"""Optional. A suggested search radius in meters. This field is used as a suggestion to the search. The actual search radius may be lower than the suggested radius in dense urban areas, and higher in regions of less business density. If the specified value is too large, a AREA_TOO_LARGE error may be returned. The max value is 40000 meters (about 25 miles)."""

Comment (bhgottfried) [reply frontend/src/FindQuiz.js:152]: Good call. I'll fix that in the next commit.

**Review by EliasTalcott — APPROVED** [review]
Functions as expected. Exciting new features :D

**Review by jhyndma — APPROVED** [review]
Functions as advertised. Re-Approved

**Pull request merged**